darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -323,9 +313,6 @@ def __init__(
 
         # setup trainer parameters from model creation parameters
         self.trainer_params = {
-            "accelerator": accelerator,
-            "gpus": gpus,
-            "auto_select_gpus": auto_select_gpus,
```

Comment:
We still need these 3 params (including your "device" logic for >= 1.7) as we decided to always use CPU by default (see my comments below on the default params)